source/opt/types.h

```diff
@@ -231,6 +231,7 @@ class Array : public Type {
   bool IsSame(Type* that) const override;
   std::string str() const override;
   const Type* element_type() const { return element_type_; }
+  uint32_t LengthId() const { return length_id_; }
```

Comment:
It could not be lower case since otherwise "Array(Type* element_type, uint32_t length_id);" above would also result in compiler warning about shadowed variable.  I do not feel happy about either, but this feels like lesser evil than adding the underscores.